01-Plot-Raster.Rmd

```diff
@@ -113,67 +126,83 @@ plot(DSM_HARV,
 ```
 
 # Plotting Data Using Breaks
-
 Often we may want to view our data subsetted by a range of values rather than
-using a continuous color ramp. This is comparable to a "classified" map in a tool
-like ArcGIS or QGIS.
-
-To begin we want use the histogram of our data to determine where the breaks in 
-our data should go. When can use `breaks=` to tell `R` to use fewer or more breaks
-or bins in our histogram. More on that below:
+using a continuous color ramp. This is comparable to a "classified" map in a
+tool like ArcGIS or QGIS.
 
-> R Bloggers Histogram Overview: <a href="http://www.r-bloggers.com/basics-of-histograms/" target="_blank">More 
-on histograms in R</a>
+Unless there is a reason for having specific breaks, we want to use a
+histogram of our data to determine where the breaks in our data should go. We 
+can use `breaks=` to tell `R` to use fewer or more breaks or bins in our 
+histogram. 
+If we name the histogram we can then enter the name an find out much of the
```

Comment:
name and find out

Megan - i am not sure what you mean by much of the underlying information in the histogram. Do you mean we can look at the assigned break values and associated counts or something like taht?
 Ie can you be more specific? Or, remove this sentence.